openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -100,6 +100,17 @@ TCN_IMPLEMENT_CALL(jlong, SSLContext, make)(TCN_STDARGS, jint protocol, jint mod
 
     UNREFERENCED(o);
 
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L && !defined(LIBRESSL_VERSION_NUMBER)
+    // TODO this is very hacky as io.netty.handler.ssl.OpenSsl#doesSupportProtocol also uses this method to test for supported protocols. Furthermore
```

Comment:
About `SSL_CTX_set_max/min_proto_version` I´m not sure if will really need that. They even define a `SSL_OP_NO_TLSv1_3` so using the options for enabling/disabling protocols seems ok for now.